dash/dash.py

```diff
@@ -378,15 +376,29 @@ def _relative_url_path(relative_package_path='', namespace=''):
 
         srcs = []
         for resource in resources:
+            is_dynamic_resource = 'dynamic' in resource and \
+                resource.get('dynamic')
+
             if 'relative_package_path' in resource:
                 if isinstance(resource['relative_package_path'], str):
-                    srcs.append(_relative_url_path(**resource))
-                else:
-                    for rel_path in resource['relative_package_path']:
+                    self.registered_paths[resource['namespace']]\
+                        .add(resource['relative_package_path'])
+
+                    if not is_dynamic_resource:
                         srcs.append(_relative_url_path(
-                            relative_package_path=rel_path,
-                            namespace=resource['namespace']
+                            resource['relative_package_path'],
+                            resource['namespace']
                         ))
+                else:
+                    for rel_path in resource['relative_package_path']:
+                        self.registered_paths[resource['namespace']]\
```

Comment:
@rmarren1 Not sure this is considered correct (escaping with '\' in the middle of a chain (as opposed to an 'if' as I've done elsewhere)

Reply:
Do you mean using the `\`? That is standard to do